### DISCUSSION TITLE
Testing on a SX-64

### DISCUSSION THREAD

**Discussion by grimm** [frntc/RAD#41] [General]
I just got a RAD device and I'm using a PI Zero W 2 with it and it is working well on my standard C64 NTSC system.  The RAD also works in my SX-64 when I tested it with Vision Basic (which requires an REU).  But when I tried to run Sonic on it, the game would crash and lockup the system right after entering the run command.  At first I thought that maybe it was the JiffyDos ROM I have installed, but disabling it didn't have any effect.  Maybe the game is doing something different than Vision Basic?  Sonic runs perfectly on my standard C64 breadbin with the same RAD device.  Or maybe it's an issue with the SX-64 Kernel ROM?

**Comment by frntc** [comment]
Hm, I don't think that it's a ROM issue. If the menu works fine, and vision basic does as well, I rather suspect that it's not working 100% reliable. The SX64 expansion port is known to behave a bit different (timing/signal-wise). Have you tried playing NUVIEs (only works on PAL!) or Koalavideo? This would indicate problems in DMA transfers visually... which you then could try to fix with adjusting the bus timings.

**Reply by grimm** [reply]
Thank you so much for looking at this for me.  I finally figured out the problem and I was able to get the game to run.  When you first run the game it will detect if you are on an NTSC system and ask if you want to continue.  The problem is that this is in white text and the default background color on the SX-64 is white.  So it gives you the impression that the game has locked up, all I needed to do was to enter in a 'Y' and the game started up no problem. 

This also effects the RAD as well, when you exit the menu, it will print the REU memory size on the bottom of the screen.  This is in white text so you can't see it on the SX-64 either.  Thanks again for your help.

**Reply by frntc** [reply]
The white background -- good point, haven't thought of this :-) (unfortunately I don't have a SX64, so I can only forward experiences from others but not test things myself)

**Reply by grimm** [reply]
If you need to test anything on the SX-64, let me know.  I would be happy to test for you.

**Comment by nik99999-s** [comment]
![Uploading nsaiq_!3123213_e921489723948148763289.png…]()

**Reply by frntc** [reply]
nik99999-s has been blocked, because of posting ads in the discussion
